src/praisonai/praisonai/cli/features/doctor/handler.py

```
@@ -297,6 +310,129 @@ def _get_categories_for_subcommand(self, subcommand: Optional[str]) -> Optional[
+                        # Write back the file
+                        with open(yaml_file, 'w') as f:
+                            yaml.dump(migrated_data, f, default_flow_style=False)
+                        
+                        files_modified += 1
+                        if not config.quiet:
+                            print(f"✅ Migrated: {yaml_file}")
+                    else:
+                        if not config.quiet:
+                            print(f"   Run with --execute to apply migration")
+                            
+            except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)